.chloggen/update-graphql-span-name.yaml

```diff
@@ -0,0 +1,4 @@
+change_type: breaking
```

Comment:
Since the GraphQL spans page is experimental, I'm not sure if this should be a `breaking` change or an `enhancement`.

Reply:
it's an enhancement and also a breaking change :)

Let's keep `breaking` - it's important to announce them.